### ISSUE TITLE
Site Executive Search Missing Accessibility in Search Tabs

### ISSUE THREAD

**Issue by martypowell** [baltimorecounty/BCPL-assets#256]
Compare the Polaris Search Tabs for the proper markup

**Comment by mxsnyder** [comment]
Resolved via SE. 

@martypowell Please have a look and close out.

**Comment by mxsnyder** [comment]
@danfox01 Please verify and close out.

**Comment by martypowell** [comment]
@mxsnyder @danfox01 i checked and it looked good to me.
